src/content/en/fundamentals/security/credential-management/retrieve-credentials.md

```diff
@@ -183,9 +183,19 @@ as part of getting credential information:
 ### Get credential information and show account chooser
 
 Show an account chooser in response to a defined user action,
-for example, when the user taps the "Sign-In" button.
-Call [`navigator.credentials.get()`](https://developer.mozilla.org/en-US/docs/Web/API/CredentialsContainer/get),
-and add `unmediated: false`, to show the account chooser:
+for example, when the user taps the "Sign-In" button. Call
+[`navigator.credentials.get()`](https://developer.mozilla.org/en-US/docs/Web/API/CredentialsContainer/get),
+and add `mediation: 'optional'` or `mediation: 'required'` to show the account chooser.
+
+When mediation is required, the user is always shown an account chooser to sign in.
```

Comment:
Let's use &lt;code&gt; tag:
When `mediation` is `required`, the user is always shown an account chooser to sign in.